app/controllers/comments_controller.rb

```diff
@@ -1,18 +1,21 @@
 class CommentsController < ApplicationController
   def new
     @post = Post.find(params[:post_id])
+    @user = @post.author
     @comment = Comment.new
   end
 
   def create
     @post = Post.find(params[:post_id])
     @comment = Comment.new(comment_params.merge(post: @post))
+    @comment.author = @post.author
 
```

Comment:
- [ ] Nice work implementing the create action for the comment :clap: However, according to the project requirement comment should be created on behalf of the `current_user` (using the method that you created in your ApplicationController). Therefore I suggest that you update the comment's author to use the `current_user` method created in the ApplicationController